### ISSUE TITLE
vue serve invalid arg type error

### ISSUE THREAD

**Issue by memic84** [vuejs/vue-cli#4237]
### Version
3.9.1

### Environment info
```
Environment Info:

  System:
    OS: macOS High Sierra 10.13.6
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
  Binaries:
    Node: 10.16.0 - ~/.nvm/versions/node/v10.16.0/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v10.16.0/bin/npm
  Browsers:
    Chrome: 75.0.3770.100
    Firefox: 63.0.3
    Safari: 12.1.1
  npmGlobalPackages:
    @vue/cli: 3.9.1
```

After some attempts, and re-installing.. this i my new env info. Issue still exists.

```
Environment Info:

  System:
    OS: macOS High Sierra 10.13.6
    CPU: (8) x64 Intel(R) Core(TM) i7-4770HQ CPU @ 2.20GHz
  Binaries:
    Node: 12.5.0 - /usr/local/bin/node
    Yarn: 1.16.0 - /usr/local/bin/yarn
    npm: 6.9.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 75.0.3770.100
    Firefox: 63.0.3
    Safari: 12.1.1
  npmPackages:
    @vue/component-compiler:  4.0.0
    @vue/component-compiler-utils:  3.0.0
    rollup-plugin-vue: ^5.0.0 => 5.0.1
    vue: ^2.6.10 => 2.6.10
    vue-runtime-helpers:  1.0.1
    vue-template-compiler: ^2.6.10 => 2.6.10
    vue-template-es2015-compiler:  1.9.1
  npmGlobalPackages:
    @vue/cli: 3.9.1
```

### Steps to reproduce
With vue-cli globally installed, and @vue/cli-service-global installed. 

1) On any vue component `vue serve myComponent.vue`

This will result in an error.

### What is expected?
Expected is that the server starts with the defaults.

### What is actually happening?
```bash
 INFO  Starting development server...
(node:46916) UnhandledPromiseRejectionWarning: TypeError [ERR_INVALID_ARG_TYPE]: The "path" argument must be of type string. Received type undefined
    at validateString (internal/validators.js:125:11)
    at Object.dirname (path.js:1260:5)
    at api.chainWebpack.webpackConfig (/Users/username/.nvm/versions/node/v10.16.0/lib/node_modules/@vue/cli-service-global/node_modules/@vue/cli-plugin-eslint/index.js:58:32)
    at webpackChainFns.forEach.fn (/Users/username/.nvm/versions/node/v10.16.0/lib/node_modules/@vue/cli-service-global/node_modules/@vue/cli-service/lib/Service.js:227:40)
    at Array.forEach (<anonymous>)
    at Service.resolveChainableWebpackConfig (/Users/username/.nvm/versions/node/v10.16.0/lib/node_modules/@vue/cli-service-global/node_modules/@vue/cli-service/lib/Service.js:227:26)
    at Service.resolveWebpackConfig (/Users/username/.nvm/versions/node/v10.16.0/lib/node_modules/@vue/cli-service-global/node_modules/@vue/cli-service/lib/Service.js:231:48)
    at PluginAPI.resolveWebpackConfig (/Users/username/.nvm/versions/node/v10.16.0/lib/node_modules/@vue/cli-service-global/node_modules/@vue/cli-service/lib/PluginAPI.js:138:25)
    at serve (/Users/username/.nvm/versions/node/v10.16.0/lib/node_modules/@vue/cli-service-global/node_modules/@vue/cli-service/lib/commands/serve.js:48:31)
    at Service.run (/Users/username/.nvm/versions/node/v10.16.0/lib/node_modules/@vue/cli-service-global/node_modules/@vue/cli-service/lib/Service.js:221:12)
(node:46916) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
(node:46916) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

---
I have tried the following:
1) Node version 8.16
2) Removing all of Node and npm, and reinstalling
3) using older versions of vue-cli (3.8) and the alpha version.

**Comment by qq326943819** [comment]
I also met, looking forward to solving.

**Comment by ricardo-lobo** [comment]
Same here, tried moments ago to use this and no success

>  Binaries:
    Node: 11.1.0 - /usr/local/bin/node
    Yarn: Not Found
    npm: 6.9.0 - /usr/local/bin/npm
  Browsers:
    Chrome: 75.0.3770.100
    Firefox: 67.0.4
    Safari: 12.1.1
  npmGlobalPackages:
    @vue/cli: 3.9.1

**Comment by sodatea** [comment]
Fixed in 3.9.2 and 4.0.0-alpha.4

**Comment by memic84** [comment]
After upgrading the package, it works! Thanks
